pkgs/top-level/python-packages.nix

```diff
@@ -6196,6 +6196,26 @@ in {
     };
   };
 
+  prov = buildPythonPackage rec {
```

Comment:
Please do not add new packages directly to this file, but store them in `python-packages` and only reference them here
as described in the header of this file.

Reply:
Thanks, can do